### PR TITLE
Update tidb-lightning-configuration.md

### PR DESCRIPTION
**Pull request by pepezzzz**
fix mistake

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

configuration mistake

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-web-interface#tidb-lightning-web-%E7%95%8C%E9%9D%A2

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ TiDB Lightning 的配置文件分为“全局”和“任务”两种类别，
...
 
 [lightning]
-# 用于拉取 web 界面和 Prometheus 监控项的 HTTP 端口。设置为 0 时为禁用状态。
+# 仅用于展示进度的 web 界面和拉取 Prometheus 监控项的 HTTP 端口。设置为 0 时为禁用状态。
 status-addr = ':8289'
 
 # 切换为服务器模式并使用 web 界面
 # 详情参见“TiDB Lightning web 界面”文档
-server-mode = false
+server-mode = true
```
Comment (sunzhaoyang) [inline tidb-lightning/tidb-lightning-configuration.md:26]: server-mode's default value is true? @gozssky

Comment (gozssky) [reply tidb-lightning/tidb-lightning-configuration.md:26]: no

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -18,12 +18,12 @@ TiDB Lightning 的配置文件分为“全局”和“任务”两种类别，
 ### tidb-lightning 全局配置
 
 [lightning]
-# 用于拉取 web 界面和 Prometheus 监控项的 HTTP 端口。设置为 0 时为禁用状态。
+# 仅用于展示进度的 web 界面和拉取 Prometheus 监控项的 HTTP 端口。设置为 0 时为禁用状态。
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-configuration.md:21]: 这个地址还会用于在服务器模式下提交任务，现在它的类型是不是整数，是一个字符串。

在 lighting 代码里的这个描述会更准确一些。 https://github.com/pingcap/tidb/blob/e10ad280572a5d28fd8fa4b698812789931f5f6d/br/tidb-lightning.toml#L4-L7

File: tidb-lightning/tidb-lightning-configuration.md (hunk 1 above)
Comment (pepezzzz) [reply tidb-lightning/tidb-lightning-configuration.md:26]: 样例上的 server-mode = false 为不会切换为服务器模式并使用 web 界面

**Review by sunzhaoyang — APPROVED** [review]
LGTM

**Pull request merged**